dist/focus-ring.js

```diff
@@ -265,7 +225,6 @@ document.addEventListener('DOMContentLoaded', function() {
     if (index(el).contains('focus-ring'))
       return;
     index(el).add('focus-ring');
-    el.setAttribute('data-focus-ring-added', '');
```

Comment:
It disambiguates an explicit, author-added `focus-ring` class (to allow always matching `.focus-ring`) from a programmatically added one. See line 281 for where it's used. It was added in https://github.com/WICG/focus-ring/commit/8d005806af5dc98b1ee2661ff764aaeda2150018